lib/render/ParticleRenderer.cpp

```diff
@@ -110,8 +110,11 @@ int ParticleRenderer::_paintGL(bool)
 
     bool renderGlyphs = GetActiveParams()->GetValueLong(ParticleParams::Render3DTag, true);
     if (renderGlyphs) {
-        if (regenerateParticles) _generateParticleGlyphs(grid, vecGrids);
-        _renderParticleGlyphs();
+        if (regenerateParticles) {
+            _generateParticleGlyphs(grid, vecGrids);
+            _renderParticleGlyphs();
+        }
+        _renderParticlesHelper();
```

Comment:
It appears `_renderParticleGlyphs()` no longer renders the particles but is only part of caching them. Could you change the name to reflect this?